src/main/java/com/suse/salt/netapi/calls/modules/Cmd.java

```diff
@@ -29,6 +31,13 @@ private Cmd() { }
                 new TypeToken<String>(){});
     }
 
+    public static LocalCall<CmdArtifacts> runAll(String cmd) {
+        LinkedHashMap<String, Object> args = new LinkedHashMap<>();
```

Comment:
i think we could replace this in all methods with `Collections.emptyMap`.

Reply:
Replaced empty map creation with ```Collections.emptyMap()```, and also replaced single pair map creations with ```Collections.singletonMap```.